Mathlib/FieldTheory/SplittingField/Construction.lean

```diff
@@ -157,7 +168,6 @@ section LiftInstances
 #noalign polynomial.splitting_field_aux.Field
 #noalign polynomial.splitting_field_aux.inhabited
 #noalign polynomial.splitting_field_aux.mk_hom
```

Comment:
Should these be removed?

Reply:
OK I am stupid, of course they should be removed now.